pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -218,17 +230,14 @@ mod pallet {
                     break;
                 }
 
-                // get the front of the queue.
-                let (who, amount) = Self::unstake_queue()
-                    .first()
-                    .expect("Has been checked before.")
-                    .clone();
+                // Get the front of the queue.
+                let (who, amount) = &Self::unstake_queue()[0];
```

Comment:
`first` is a more meaningful statement. You could remove `clone`  if you just want to use reference.

Reply:
that will meet rust compiler issue